draft-ietf-oauth-transaction-tokens.md

```diff
@@ -433,6 +437,19 @@ grant_type=urn%3Aietf%3Aparams%3Aoauth%3Agrant-type%3Atoken-exchange
 ~~~
 {: #figtxtokenrequest title="Example: Txn-Token Request"}
 
+### Subject Token Types
```

Comment:
You might want to reconcile these with how Token Exchange (not particularity well) defines subject_token_type   https://datatracker.ietf.org/doc/html/rfc8693#section-2.1-4.14 and points to https://datatracker.ietf.org/doc/html/rfc8693#TokenTypeIdentifiers about identifiers

Reply:
Oh and note Aaron's draft use of the `urn:ietf:params:oauth:token-type:id_token` URI for id tokens https://www.ietf.org/archive/id/draft-parecki-oauth-identity-assertion-authz-grant-00.html#section-5

Reply:
I agree with Brian. I think we need to make these URIs. If we want to explicitly support an rfc9068 token then we should define a URI for it. However, if the intent of the `rfc9068` value is to represent an access_token then we should use the value defined within RFC 8693.

Reply:
Thanks @bc-pi , @gffletch and @yaronf for your feedback here. I've updated the PR. Please take a look. thanks